overlays/default-modules.nix

```diff
@@ -0,0 +1,22 @@
+# This overlay adds `defaultModules` to work around issues with
```

Comment:
Further random poking around: is this not the same in function as `modules/configuration-nix.nix`?